build_ngtf.py

```diff
@@ -512,6 +512,7 @@ def main():
         "-DNGRAPH_TUNE_ARCH=" + target_arch,
         "-DNGRAPH_ARTIFACTS_DIR=" + artifacts_location,
         "-DUNIT_TEST_ENABLE=ON",
+        "-DNGRAPH_DISTRIBUTED_ENABLE=TRUE",
```

Comment:
The default should be false i.e., for regular builds no need to enable distributed. Need to add a command line option to the build_ngtf.py (e.g., `--enable_distributed`) which when specified will add this flag. See the --debug_build as an example of how to extend the cmake options.

Reply:
Added the option of `--distributed_build` for distributed build